City-of-Helsinki/drupal-infofinland
# UHF-10075 GDPR-module removal

# [UHF-10075](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10075)
- Remove the questionable gdpr-module implementation
- Disabled dblog-module.
- Automatically delete the log file with update hook.
- Removed other obsolete code (if_migration & select2 -placeholder module)

## How to install

* **If you don't already have it, add the log file to correct place to test the deletion**
  * public/sites/default/files/logs
  * filename: infofinland_gdpr.log 
* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10075`
  * `make fresh`
* Run `make drush-cr`

## How to test
Go check that the log file was deleted
The site should work normally.
You should be able to view the users normally.
No errors should occur while browsing the site and admin-side


## Continuous documentation



## Translations



## Other PRs


* Link to other PR


[UHF-10075]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10075?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
